tests/mock/test_topas_minimal.py

```diff
@@ -0,0 +1,92 @@
+"""Test topas mock"""
+
+import logging
+import os
+from pathlib import Path
+from subprocess import call
+from typing import Generator
+import numpy as np
+
+import pytest
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='module')
+def topas_mock() -> Generator[Path, None, None]:
+    """Return path to mock script"""
+    yield Path(__file__).parent.parent / "res" / "mocks" / "topas_minimal" / "topas"
+
+
+@pytest.fixture(scope='module')
+def expected_results() -> Generator[dict, None, None]:
+    """Return expected result"""
+    yield {
+        "fluence_bp_protons_xy.csv": {
+            "shape": [1, 4, 4],
+            "4x4": [
+                [0.00177, 0., 0., 0.00088],
+                [0.00088, 0.00177, 0., 0.00088],
+                [0.00266, 0.00088, 0.00177, 0.00177],
+                [0.00088, 0., 0., 0.]
+            ]
+        },
+        "fluence_bp_protons_xy2.csv": {
+            "shape": [1, 4, 4],
+            "4x4": [
+                [0.00177, 0., 0., 0.00088],
+                [0., 0.00177, 0.00177, 0.00177],
+                [0.00266, 0.00088, 0.00088, 0.],
+                [0., 0., 0., 0.]
+            ]
+        }
+    }
+
+
+@pytest.mark.skipif(os.name == 'nt', reason="Windows not supported")
+@pytest.mark.parametrize('output_file', ["fluence_bp_protons_xy.csv", "fluence_bp_protons_xy2.csv"])
+def test_topas_mock(tmp_path: Path, output_file: str, expected_results: dict, topas_mock: Path):
+    """Test topas mock script"""
+    assert topas_mock.exists(), "Topas script does not exist"
+
+    env = append_path_to_environ(topas_mock.parent)
+    call("topas", cwd=tmp_path, env=env)  # skipcq: BAN-B607
+    output_files = [f.name for f in list(tmp_path.glob("*"))]
```

Comment:
```suggestion
    output_files = [f.name for f in list(tmp_path.glob("*.csv"))]
```

Shouldn't we go through CSV files only ?